### ISSUE TITLE
IE11 supporting

### ISSUE THREAD

**Issue by ldtung** [David-Mulder/paper-datatable#75]
Hi, do you plan to support IE11? Or any workaround for me to make this component working on IE11?

**Comment by aitoroses** [comment]
+1

**Comment by jahglow** [comment]
+100 I'd include ability to support IE10+ as Polymer lib would.

**Comment by allenmoatallen** [comment]
+1

**Comment by dam0vm3nt** [comment]
Probably the problem is 

https://github.com/Polymer/polymer/issues/1735

Also this block `dart` porting.

Will you consider avoid using `<table>` in `paper-datatable` ?

**Comment by maxiplay** [comment]
+1 I have to choose an other table solution.. Be aware of the current market you loose people

**Comment by maxiplay** [comment]
For people with same problem than me? I advise to use https://github.com/Saulis/iron-data-table

**Comment by dam0vm3nt** [comment]
@maxiplay I don't think you can talk about a "market" in a open-source free component library context.

@David-Mulder did a great job 'til now and I hope he will fix this, meanwhile `iron-data-table` is a more than good replacement and by now almost all the features of `paper-datatable` are replicated so to me that is just enough.

**Comment by maxiplay** [comment]
Sorry for misanderstanding, I speak about browser market for developper like me. And I don't questioning the quality of paper-datatable, rightly, I expose my frustration because I like this tool. But because a point of view on IE11 and safari, this good tool loose people whereas it does not deserve it.
